debug/customization/vector-tile-layer/smk-init.js

```diff
@@ -0,0 +1,298 @@
+                    transit: {
+                        weight: 0.5,
+                        fillColor: '#f2b648',
+                        color: '#f2b648',
+                        fillOpacity: 0.2,
+                        opacity: 0.4,
+                    },
+                    building: {
+                        fill: true,
+                        weight: 1,
+                        fillColor: '#2b2b2b',
+                        color: '#2b2b2b',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    water_name: {
+                        weight: 1,
+                        fillColor: '#022c5b',
+                        color: '#022c5b',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    transportation_name: {
+                        weight: 1,
+                        fillColor: '#bc6b38',
+                        color: '#bc6b38',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    place: {
+                        weight: 1,
+                        fillColor: '#f20e93',
+                        color: '#f20e93',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    housenumber: {
+                        weight: 1,
+                        fillColor: '#ef4c8b',
+                        color: '#ef4c8b',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    poi: {
+                        weight: 1,
+                        fillColor: '#3bb50a',
+                        color: '#3bb50a',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    earth: {
+                        fill: true,
+                        weight: 1,
+                        fillColor: '#c0c0c0',
+                        color: '#c0c0c0',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    }
+                };
+
+                // Monkey-patch some properties for nextzen layer names, because
+                // instead of "building" the data layer is called "buildings" and so on
+                vectorTileStyling.buildings  = vectorTileStyling.building;
+                vectorTileStyling.boundaries = vectorTileStyling.boundary;
+                vectorTileStyling.places     = vectorTileStyling.place;
+                vectorTileStyling.pois       = vectorTileStyling.poi;
+                vectorTileStyling.roads      = vectorTileStyling.road;
+
+            	// Nextzen tiles PBF layer setup
+
+                // Assumes layers = "all", and format = "mvt"
+                var nextzenTilesUrl = "https://tile.nextzen.org/tilezen/vector/v1/512/all/{z}/{x}/{y}.mvt?api_key={apikey}";
```

Comment:
We shouldn't be using var.

Reply:
Choosing between let and const actually shouldn't depend on the scope; let should only be used for variables that have a value re-assigned to them and const for everything else.

Reply:
I think every instance of 'let' in this file can actually be replaced with 'const'.

Reply:
I confirmed we can replace all 'let's with 'const's since we don't reassign values to any of the 'let's - we only update objects or arrays.